shell/platform/embedder/embedder_render_target.cc

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/embedder/embedder_render_target.h"
+
+#include "flutter/fml/logging.h"
+
+namespace flutter {
+
+EmbedderRenderTarget::EmbedderRenderTarget(FlutterBackingStore backing_store,
+                                           sk_sp<SkSurface> render_surface,
+                                           fml::closure on_release)
+    : backing_store_(backing_store),
+      render_surface_(std::move(render_surface)),
+      on_release_(on_release) {
+  // The optimization to elide backing store updates between frames has not been
+  // implemented yet.
```

Comment:
Let's file an issue describing what has to be done and link it from a TODO here.

Reply:
Done.